sklearn/preprocessing/imputation.py

```diff
@@ -177,6 +178,10 @@ def fit(self, X, y=None):
                                                    self.strategy,
                                                    self.missing_values,
                                                    self.axis)
+            invalid_mask = np.isnan(self.statistics_)
```

Comment:
It isn't explicitly mentioned anywhere but if the column has all missing values, the corresponding element in ``statistics_`` is ``nan``.